### DISCUSSION TITLE
Application launching microsoft store

### DISCUSSION THREAD

**Discussion by Otsdarvatheeverchosen** [Shazbot/WH3-Mod-Manager#60] [Q&A]
For some reason when I try to launch the exe it just brings up microsoft store and asks me if I wanna search for an app.

**Comment by JoenvandeVorle** [comment]
Only sorta related, but I used to play via Game Pass and launching the game via Xbox app sometimes started the Epic Games store instead. Not sure what happens there but my guess is the problem is on the game side and not the mod manager side

**Reply by Otsdarvatheeverchosen** [reply]
Yeah but the game launches fine without the mod manager so idk how to figure out the problem.

**Reply by Otsdarvatheeverchosen** [reply]
Nvm I fixed it, turns out using windows crappy zip files and switching it to 7zip somehow fixed it.
